Packs/Wiz/IncidentFields/incidentfield-Wiz_Resource_Native_Type.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
@solalraveh what is the difference between `Wiz Resource Native Type` and `Wiz Resource Type`?

Reply:
See PR description above. 
One is the native type from the cloud provider, other is the Wiz type which used in one of the commands. 
It was split to avoid confusion, with examples given in field type and RN.